### DISCUSSION TITLE
Blog comment on The Details Behind the Sphere Dataset in Weaviate

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#856] [Docs/blog comments]
# Blog comment on The Details Behind the Sphere Dataset in Weaviate

Learn about the hardware, software and performance metric specifications behind our ~1B object import of the Sphere dataset into Weaviate.

https://weaviate.io/blog/details-behind-the-sphere-dataset-in-weaviate

**Comment by lqhl** [comment]
Sounds great. I'm wondering how much RAM I need to hold the dataset in Weaviate. Do you have a recommended hardware spec for 1M/10M/100M/899M lines dataset

**Comment by oppiet30** [comment]
If I wanted to run weaviate in a docker container to do this, can I do it?
Can someone add the docker-compse.xml file so I can do this without doing it on weavaite's cloud instance?
